fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -487,6 +480,7 @@ struct PhysicalShard {
 	std::unordered_map<std::string, std::unique_ptr<DataShard>> dataShards;
 	std::shared_ptr<ReadIteratorPool> readIterPool;
 	bool deletePending = false;
+	std::atomic<bool> inited;
```

Comment:
`initialized`

Reply:
The compiler doesn't allow member variable and member function share the same name, renamed it as `isInitialized`.